.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   global:
     - VERSION=1.0
     - NETWORK=mainnet
-    - CARDANO_SL_BRANCH=cardano-sl-1.0
+    - CARDANO_SL_BRANCH=release/1.0.4
```

Comment:
This should be `master`.

Reply:
Also it should always be in sync with appveyor

Reply:
Done..